### PR TITLE
zhang19 gaia names

### LINKED ISSUE #245: Fix Zhang19 source names
- Gaia J0452-36A should be Gaia EDR3 4818823636756117504
- Gaia J0452-36B should be Gaia EDR3 4818823808553134592
The Gaia J names should be removed.

### PR DESCRIPTION
**Pull request by LishaRamon**
Short description: Changing two Gaia names

Gaia J0452-36A should be Gaia EDR3 4818823636756117504
Gaia J0452-36B should be Gaia EDR3 4818823808553134592
The Gaia J names should be removed.

Link to relevant issue: Closes #245

For data ingests:
- [x] includes modified JSON files
- [x] Update the Versions table

### CODE REVIEW

**Comment by LishaRamon** [comment]
There is no gaia_j0452-36a json file found to modify.

**Comment by kelle** [comment]
Then I guess we should ingest it.  

Here's the original paper.
https://academic.oup.com/mnras/article/489/1/1423/5545604

**Comment by LishaRamon** [comment]
How can I run/check that the json file edit was implemented in the db?

**Comment by LishaRamon** [comment]
> Then I guess we should ingest it.
> 
> Here's the original paper. https://academic.oup.com/mnras/article/489/1/1423/5545604

Ingest using a new file or is there an existing file to add to?

**Comment by kelle** [comment]
I'm not sure I understand the question. The JSON file modification is changing the database. If you want to see the change in the change in the DB browser, you reload the database. That will generate a new SQLite file using the JSON files.

**Comment by LishaRamon** [comment]
That's what I thought too and reloaded the db. Unless I misunderstood what you meant by changing the names. I added 'Gaia EDR3 4818823808553134592' as another name under 'Gaia J0452-36B'. Or does it mean to delete the 'Gaia J0452-36B.json' file and create a new one with the name 'Gaia EDR3 4818823808553134592'?

**Comment by LishaRamon** [comment]
> Then I guess we should ingest it.
> 
> Here's the original paper. https://academic.oup.com/mnras/article/489/1/1423/5545604

This paper is already ingested. Is gaia_j0452-36a and gaia_j0452-36b supposed to be two sources from the same paper? 36b is there but no json for 36a.

**Comment by kelle** [comment]
It might have gotten changed back when you merged. The way you had it originally was correct image

**Comment by kelle** [comment]
Yes, two sources from the same paper.

**Comment by LishaRamon** [comment]
It does not pass the test when changing the source because it says there is no Gaia EDR3 4818823808553134592 in the Source table, just the first name.

**Comment by LishaRamon** [comment]
I attempted making a json file for gaia 36a, but the ra, dec, equinox, and spectral type code are wrong. I only got the spectral type string and changed the source name. In the paper it seems like I have to convert the coordinates data into decimal before I can ingest them.

**Comment by LishaRamon** [comment]
Table 1 Data

Table 1 Screenshot for Parallax + ProperMotion Data

**Comment by kelle** [comment]
- The separation in arcseconds (115.3) goes in the Companion Relationships table.
- you could also add the T_eff, metallicty (Fe/H), gravity (logg), and mass to the Modeled Parameters table.

**Comment by kelle** [comment]
Might as well also do the photometry. The relevant filters names from the Photometry Filters table are: 
- Gaia2.G
- Gaia2.Gbp
- Gaia2.Grp
- VHS J = VISTA.J
- VHS K = VISTA.K_s
- WISE.W1
- WISE.W2

File: data/gaia_j0452-36b.json [hunk 1]
```diff
@@ -15,14 +15,48 @@
...
+            "value_error": 0.0014,
+            "unit": "M_jup",
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "parameter": "metallicity",
+            "value": -1.4,
+            "value_error": 0.2,
+            "unit": "Fe/H",
```
Comment (kelle) [inline data/gaia_j0452-36b.json:55]: unit should be "dex"
```suggestion
            "unit": "dex",
```

File: data/gaia_j0452-36b.json [hunk 2]
```diff
@@ -15,14 +15,48 @@
...
+            "comments": null,
```
Comment (kelle) [inline data/gaia_j0452-36b.json:56]:
```suggestion
            "comments": "[Fe/H]",
```

File: data/gaia_j0452-36b.json [hunk 3]
```diff
@@ -15,14 +15,48 @@
...
+            "value_error": 0.2,
+            "unit": "dex",
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "parameter": "mass",
+            "value": 0.0855,
+            "value_error": 0.0014,
+            "unit": "M_jup",
```
Comment (kelle) [inline data/gaia_j0452-36b.json:47]: Be super careful here.  The paper provides these as solar masses.  The circle with the dot in it means the Sun.  
```suggestion
            "unit": "M_sun",
```

File: data/gaia_j0452-36b.json [hunk 4]
```diff
@@ -12,11 +12,144 @@
...
+            "telescope": "Gaia",
+            "epoch": null,
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "band": "VISTA.J",
+            "magnitude": 16.437,
+            "magnitude_error": 0.008,
+            "telescope": "Gaia",
```
Comment (kelle) [inline data/gaia_j0452-36b.json:109]:
```suggestion
            "telescope": "VISTA",
```

File: data/gaia_j0452-36b.json [hunk 5]
```diff
@@ -12,11 +12,144 @@
...
+            "epoch": null,
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "band": "VISTA.K_s",
+            "magnitude": 15.954,
+            "magnitude_error": 0.038,
+            "telescope": "Gaia",
```
Comment (kelle) [inline data/gaia_j0452-36b.json:118]:
```suggestion
            "telescope": "VISTA",
```

File: data/gaia_j0452-36b.json [hunk 6]
```diff
@@ -12,11 +12,144 @@
...
+            "epoch": null,
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "band": "WISE.W1",
+            "magnitude": 15.571,
+            "magnitude_error": 0.037,
+            "telescope": "Gaia",
```
Comment (kelle) [inline data/gaia_j0452-36b.json:127]:
```suggestion
            "telescope": "WISE",
```

File: data/gaia_j0452-36b.json [hunk 7]
```diff
@@ -12,11 +12,144 @@
...
+            "epoch": null,
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "band": "WISE.W2",
+            "magnitude": 15.234,
+            "magnitude_error": 0.068,
+            "telescope": "Gaia",
```
Comment (kelle) [inline data/gaia_j0452-36b.json:136]:
```suggestion
            "telescope": "WISE",
```

**Comment by LishaRamon** [comment]
Tests for data and integrity are failing completely at the moment

**Review by kelle — APPROVED** [review]
Looks great!

**Pull request merged**